src/librender/python/texture3d_v.cpp

```diff
@@ -1,33 +1,29 @@
-#include <mitsuba/core/properties.h>
 #include <mitsuba/render/interaction.h>
-#include <mitsuba/render/spectrum.h>
+#include <mitsuba/render/texture.h>
 #include <mitsuba/python/python.h>
 
 MTS_PY_EXPORT(Volume) {
```

Comment:
Let's move all of this this to the `texture_v.cpp` file.

Reply:
Done!